etc/skate-style/labels.mss

```diff
@@ -486,7 +556,6 @@
   text-spacing: 100;
   text-size:11;
   text-clip: false;
-  text-min-padding: 5;
```

Comment:
This seems to be the culprit of the issue design raised [here](https://app.asana.com/0/0/1203773677774396/1203773716164185/f) where some streets at low-levels were missing.

This was avoiding placing text [near the edge of the tiles](https://cartocss.readthedocs.io/en/latest/mapnik_api.html#text-min-padding), but because we use a buffer-size to properly render text across edges, this is not necessary.

There are other locations we are using this setting that we probably won't need to. This was hard to test for locally though, so my plan is to check back in with design after the new tiles are generated to see if we should make this change more widespread.